accounts/abi/argument.go

```diff
@@ -33,6 +33,7 @@ func (a *Argument) UnmarshalJSON(data []byte) error {
 	var extarg struct {
 		Name string
 		Type string
+		Indexed bool
```

Comment:
This is correct because not all argument may be indexed. It mirrors [the `indexed` modifier keyword in solidity](http://solidity.readthedocs.io/en/develop/contracts.html?highlight=indexed#events).